internal/cmd/workspaces.go

```diff
@@ -124,6 +132,7 @@ func lsWorkspacesCommand() *cobra.Command {
 		},
 	}
 
+	cmd.Flags().BoolVar(&all, "all", false, "Get workspaces for all users")
```

Comment:
```suggestion
	cmd.Flags().BoolVar(&all, "all", false, "Get workspaces for all users (admin only)")
```

Only admins can get all workspaces by workspace provider it seems